package-lock.json

```diff
@@ -3459,15 +3459,6 @@
       "resolved": "https://registry.npmjs.org/binary-extensions/-/binary-extensions-2.1.0.tgz",
       "integrity": "sha512-1Yj8h9Q+QDF5FzhMs/c9+6UntbD5MkRfRwac8DoEm9ZfUBZ7tZ55YcGVAzEe4bXsdQHEk+s9S5wsOKVdZrw0tQ=="
     },
-    "bindings": {
-      "version": "1.5.0",
-      "resolved": "https://registry.npmjs.org/bindings/-/bindings-1.5.0.tgz",
-      "integrity": "sha512-p2q/t/mhvuOj/UeLlV6566GD/guowlr0hHxClI0W9m7MWYkL1F0hLo+0Aexs9HSPCtR1SXQ0TD3MMKrXZajbiQ==",
-      "optional": true,
-      "requires": {
-        "file-uri-to-path": "1.0.0"
-      }
-    },
```

Comment:
What happened here? Why are these getting removed?

Reply:
I think this is from when I was troubleshooting docker. I don't see "file-to-uri-path" or "bindings" in package.json so they must've not installed on the fresh npm install after removing the package-lock. I'm curious because there's no merge error saying I've removed anything from package.json itself. So I'm lost as to where the fluke is. Were these dependencies two dependencies removed at some point?